src/components/Utils.js

```diff
@@ -6,6 +6,22 @@ import { STATUS, ACTIONS } from "react-joyride";
 import { NONE } from "./Pages/Widgets/MELightDropDown";
 import { COPYRIGHT_OPTIONS } from "./Constants";
 
+
+export function domainsAreTheSame(url1, url2) {
+  try {
+      let parsedUrl1 = new URL(url1);
+      let parsedUrl2 = new URL(url2);
+
+      // Remove 'www.' prefix if present
+      let domain1 = parsedUrl1.hostname.replace(/^www\./, '');
+      let domain2 = parsedUrl2.hostname.replace(/^www\./, '');
+
+      return domain1 === domain2;
```

Comment:
question: is there ever a lowercase/uppercase issue to worry about?  If so could convert

Reply:
Looks like my browser converts to lowercase, but just want to be sure in general

Reply:
Good observation @BradHN1 , but we do not need to worry about lower case... At least not for the hostname. When we convert the url to the JS URL object, it converts the hostname & host to lowercase... only the pathname is left as is.